src/Lean/Server/FileWorker.lean

```diff
@@ -401,7 +401,7 @@ section MessageHandling
       catch e =>
         ctx.hOut.writeLspResponseError
           { id
-            code := ErrorCode.internalError
+            code := ErrorCode.workerCrashed
```

Comment:
The worker does not crash at this point, it only signals that the given request has failed. We could have an RPC-specific code here, but that's probably orthogonal to the PR so let's leave it as `internalError`.